server/models/Event.js

```diff
@@ -24,4 +27,4 @@ const eventSchema = new Schema({
 
 const Event = model("Event", eventSchema);
 
-module.exports = Event;
+module.exports = eventSchema;
```

Comment:
I think we'll want to export the model here instead of the schema so that we can define routes based on the Event model.